docs/core/compatibility/core-libraries/9.0/ambiguous-overload.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Breaking change: Ambiguous overload resolution affecting StringValues implicit operators"
+description: Learn about the .NET 9 breaking change in core .NET libraries where ambiguous overload resolution now throws error CS0121.
+ms.date: 02/24/2025
+ai-usage: ai-assisted
+---
+
+# Ambiguous overload resolution affecting StringValues implicit operators
+
+In .NET 9, a breaking change in the [`params Span<T>` lang feature](../../../whats-new/dotnet-9/libraries.md#params-readonlyspant-overloads) creates ambiguity with the implicit operators of <xref:Microsoft.Extensions.Primitives.StringValues>. This change results in the compiler throwing error `CS0121` when it encounters ambiguous method calls.
+
+## Previous behavior
+
+The APIs mentioned in the [Affected APIs](#affected-apis) section previously had no overloads ambiguous with the implicit operators of <xref:Microsoft.Extensions.Primitives.StringValues>. As a result, the compiler would resolve the overloads without any issues.
+
+## New behavior
+
+The compiler throws error `CS0121` for ambiguous overloads. The same code will now result in the following error:
```

Comment:
What does "the same code" refer to?

Reply:
It refers to an example I removed and then forgot to update this line. 😁